### ISSUE TITLE
find-moj-data - all form controls should have labels

### ISSUE THREAD

**Issue by jemnery** [ministryofjustice/analytical-platform#3269]
For accessibility, all form controls should have an associated `<label>`

Search box - it's OK to do something like this to create a visually hidden label:

```html
<label for="searchForm-search" class="govuk-label govuk-visually-hidden-focusable">Search MOJ Data</label>
```

The checkbox labels just need a `for` attribute added:

![Image](https://github.com/ministryofjustice/data-platform/assets/1306705/4ee8badc-06ee-4d6b-94b2-6ece0818a4c1)

**Comment by murdo-moj** [comment]
https://github.com/ministryofjustice/find-moj-data/pull/73
